### PR TITLE
fix incorrect dir file classification

### PR DESCRIPTION
**Pull request by matthewgoslett**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ composer.lock
 vendor
 bin
 coverage
-coverage.xml
\ No newline at end of file
+coverage.xml
+dev
```
Comment (cedricziel) [inline .gitignore]: missing line break

**Comment by cedricziel** [comment]
Looks much better than the 0 byte check. I'm fine with it

**Comment by paulcanning** [comment]
Looks good to me! tested in my local adapter, works fine.

**Comment by matthewgoslett** [comment]
I forgot to tag the issues:
This PR fixes:
https://github.com/Superbalist/flysystem-google-storage/issues/23
https://github.com/Superbalist/flysystem-google-storage/issues/19

**Pull request merged**